bom-2.204.x/pom.xml

```diff
@@ -9,7 +9,7 @@
     <artifactId>bom-2.204.x</artifactId>
     <packaging>pom</packaging>
     <properties>
-        <configuration-as-code-plugin.version>1.36</configuration-as-code-plugin.version>
+        <configuration-as-code-plugin.version>1.36.1-beta</configuration-as-code-plugin.version>
```

Comment:
Awaiting proper release of backport

Reply:
So should this be in draft status until then? My concern is that someone on a 2.204.x BOM releases a plugin update which depends on CasC 1.36.1-beta or newer, and then their update becomes uninstallable on actual 2.204.x systems.

Would be clearer to use incremental versions for this sort of thing: https://github.com/jenkinsci/bom/pull/294#discussion_r500594614

Reply:
(still need to do [JENKINS-50804](https://issues.jenkins-ci.org/browse/JENKINS-50804))